### PR TITLE
Edit profile template

### PR DESCRIPTION
**Pull request by vuminhhieu1311**
## Related Tickets
- [#39425](https://edu-redmine.sun-asterisk.vn/issues/39425)

## WHAT (optional)
Design profile edit layout
## Evidence (Screenshot or Video)
Screen Shot 2021-08-04 at 11 19 34

### CODE REVIEW

**Review by nvvu99 — APPROVED** [review]
Ok

**Comment by vuminhhieu1311** [comment]
ready

File: resources/views/layouts/menu.blade.php [hunk 1]
```diff
@@ -195,39 +195,40 @@ class="fa fa-angle-down"></i></a>
                                             <li><a href="about.html">About-Us</a></li>
                                             <li><a href="404%20error.html">404</a></li>
                                             <li><a href="add_postin.html">Add-Posting</a></li>
-                                            <li><a href="login.html">Login</a></li>
-                                            <li><a href="register.html">Register</a></li>
-                                            <li><a href="pricing.html">Pricing</a></li>
-                                        </ul>
+                                            <li><a href="{{ route('login') }}">{{ __('messages.login') }}</a></li>
+                                            <><a href="{{ route('register') }}>{{ __('messages.register') }}</a></li>
+                                            <li><a href=" pricing.html">Pricing</a>
```
Comment (huyenttm-0901) [inline resources/views/layouts/menu.blade.php:200]: sai gì này em

**Comment by vuminhhieu1311** [comment]
ready

**Pull request merged**